### PR TITLE
feat: uitransform pointerFilter property

### PR DESCRIPTION
**Pull request by pravusjif**
Added `PointerFilterMode` in `UiTransform`.

-------------------
Related PRs:
* https://github.com/decentraland/js-sdk-toolchain/pull/792
* https://github.com/decentraland/unity-renderer/pull/5819

### CODE REVIEW

**Review by leanmendoza — APPROVED** [review]

**Comment by pravusjif** [comment]
I'll merge after I get approval in the other 2 related PRs.

**Pull request merged**
